src/components/Header/LargeHeader.styles.js

```diff
@@ -1,13 +1,13 @@
-import { css } from '@emotion/core';
+import { css } from 'lib/utils/media-queries';
 import { Color, Space } from 'lib/theme';
 
 const styles = {
   root: css({
     alignItems: 'flex-end',
     backgroundColor: `${Color.CORAL}`,
     display: 'flex',
-    minHeight: '276px',
-    padding: `${Space.S40}px`,
+    minHeight: ['276px', '', 'auto'],
```

Comment:
for an explicit px value like this, could we save it as a const at the top of the file that defines what it is?

Reply:
Got it, thanks @lizbanach!